src/scheduling/setup_lp.py

```diff
@@ -25,19 +98,34 @@ def set_up_base_lp(
         base_jobs (list[QuantumCircuit]): The list of quantum cirucits (jobs).
         accelerators (dict[str, int]): The list of available accelerators (machines).
         big_m (int): Metavariable for the LP.
-        timesteps (list[int]): Meta variable for the LP, big enough to cover largest makespan.
+        timesteps (int): Meta variable for the LP, big enough to cover largest makespan.
 
     Returns:
         LPInstance: The LP instance object.
     """
     # Set up input params
-    jobs = ["0"] + [str(idx + 1) for idx, _ in enumerate(base_jobs)]
     job_capacities = {str(idx + 1): job.num_qubits for idx, job in enumerate(base_jobs)}
     job_capacities["0"] = 0
```

Comment:
This is changing the behavior: Now the job 0 is at the end (because `jobs = list(job_capacities.keys())` in `_define_lp` uses the natural order of the dict keys, which is the insertion order). But it seems that other methods depend on the job 0 being the first, e.g. `set_up_simple_lp` which uses `lp_instance.jobs[1:]`